<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="f0711fc9-e942-4013-be12-8628a97a61f2" name="Vasya@TP-X220-2 2015-01-19 10:14:01" runUser="TP-X220-2\Vasya" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="8084e574-4d63-4013-b7c3-116ebeeb94fb">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment userDeploymentRoot="C:\Users\Vasya\Desktop\GitWeatherProject\WeatherProject\HistoryOfObservations\TestResults" useDefaultDeploymentRoot="false" enabled="false" runDeploymentRoot="Vasya_TP-X220-2 2015-01-19 10_14_01">
      <DeploymentItem filename="C:\Users\Vasya\Desktop\GitWeatherProject\WeatherProject\HistoryOfObservations\HistoryOfObservations\bin\Debug\HistoryOfObservations.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2015-01-19T10:14:01.2021867+03:00" queuing="2015-01-19T10:14:01.2177867+03:00" start="2015-01-19T10:14:01.2333867+03:00" finish="2015-01-19T10:14:01.4673872+03:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="MailProviderTest" storage="c:\users\vasya\desktop\gitweatherproject\weatherproject\historyofobservations\testproject\bin\debug\testproject.dll" id="8ce59ba4-54da-1eab-17ec-b36ff8ae2a48">
      <Execution id="a84559f8-a109-46e5-b66a-d067c8a41aab" />
      <TestMethod codeBase="C:/Users/Vasya/Desktop/GitWeatherProject/WeatherProject/HistoryOfObservations/TestProject/bin/Debug/TestProject.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject.MistakeExpectedValueAnaliserTest, TestProject, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="MailProviderTest" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="8ce59ba4-54da-1eab-17ec-b36ff8ae2a48" executionId="a84559f8-a109-46e5-b66a-d067c8a41aab" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="a84559f8-a109-46e5-b66a-d067c8a41aab" testId="8ce59ba4-54da-1eab-17ec-b36ff8ae2a48" testName="MailProviderTest" computerName="TP-X220-2" duration="00:00:00.0691265" startTime="2015-01-19T10:14:01.2333867+03:00" endTime="2015-01-19T10:14:01.4517871+03:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="a84559f8-a109-46e5-b66a-d067c8a41aab">
      <Output>
        <ErrorInfo>
          <Message>Test method TestProject.MistakeExpectedValueAnaliserTest.MailProviderTest threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.</Message>
          <StackTrace>    at HistoryOfObservations.Providers.MailRuWeatherProvider.getTempOfADayFromTempTagCollection(HtmlNodeCollection tempers, Int32 days) in C:\Users\Vasya\Desktop\GitWeatherProject\WeatherProject\HistoryOfObservations\HistoryOfObservations\Providers\MailRuWeatherProvider.cs:line 40
   at HistoryOfObservations.Providers.MailRuWeatherProvider.GetDayliObservation() in C:\Users\Vasya\Desktop\GitWeatherProject\WeatherProject\HistoryOfObservations\HistoryOfObservations\Providers\MailRuWeatherProvider.cs:line 28
   at TestProject.MistakeExpectedValueAnaliserTest.MailProviderTest() in C:\Users\Vasya\Desktop\GitWeatherProject\WeatherProject\HistoryOfObservations\TestProject\MistakeExpectedValueAnaliserTest.cs:line 65
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>